src/Status/StatusCountTable/StatusCountTable.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography
+} from "@mui/material";
+
+import React from "react";
+import { Status } from "../statuses.types";
+import { StatusCountTableProps } from "./StatusCountTable.types";
+import StatusIcon from "../StatusIcon";
+
+// custom status count table component, which will be used in the status bar component on hover to show aditional data about statuses
```

Comment:
We should use jsdoc syntax here as it then gets picked up by the storybook docs.

```suggestion
/**
 * Table that renders statuses vs. the number of items that match that status. Header row shows the total count alongside a title.
 */
```